scripts/update_versions.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+"""
+Updates versions and dates in all of our FHIR resources.
+
+Example:
+    $ scripts/update_versions.py 1.1.1
+"""
+import json
+import sys
+from datetime import datetime, timezone
+from pathlib import Path
+
+OUR_URLS = (
+    "http://fhir.kids-first.io",
+    "https://fhir.kids-first.io",
+)
+
+
+def bump_ini(filePath, new_version):
+    with filePath.open(mode="r") as f:
+        body = [
+            line
+            if not line.startswith("version ")
+            else f"version = {new_version}\n"
+            for line in f
+        ]
+    with filePath.open(mode="w") as f:
+        f.writelines(body)
+
+
+def bump_json(filePath, new_version):
+    try:
+        with filePath.open(mode="r") as f:
+            body = json.load(f)
+    except json.decoder.JSONDecodeError:
+        return
+    if body.get("url", "").startswith(OUR_URLS):
+        body["version"] = new_version
+        if "date" in body:
+            body["date"] = datetime.now().astimezone(timezone.utc).isoformat()
+        with filePath.open(mode="w") as f:
+            json.dump(body, f, indent=2)
+
+
+if __name__ == "__main__":
```

Comment:
I think this script (or CLI command if it gets moved into `fhirutil`) needs to be provided 1 input parameter, the path to the IG .ini file. From there it should be able to determine the IG site root (the directory the ini is in) and also the directory(ies) where the resources live. That's what happens in the `fhirutil validate` command. See https://github.com/ncpi-fhir/ncpi-fhir-utility/blob/d4e0eb3f2907f3455e5f535446f436f1976f2039/ncpi_fhir_utility/app.py#L72

Reply:
https://github.com/kids-first/kf-model-fhir/pull/149/commits/76f48f4049e321a895f1a3edd3495a38338396e4